app/models/customer.py

```diff
@@ -0,0 +1,28 @@
+from flask import Flask, current_app
+from flask_sqlalchemy import SQLAlchemy
+from app import db
+
+class Customer(db.Model):
+    customer_id = db.Column(db.Integer, primary_key=True)
+    name = db.Column(db.String)
+    postal_code = db.Column(db.String)
+    phone = db.Column(db.String)
+    registered_at = db.Column(db.DateTime)
+    videos_checked_out_count = db.Column(db.Integer, default=0)
+
```

Comment:
You can also add relationships so that you can access things more easily.

```python
    rentals = db.relationship("Rental", backref="customer", lazy='dynamic')

```

Then you can do things like, `my_customer.rentals` to get a list of rentals for a customer.